data/json/itemgroups/SUS/trash.json

```
@@ -233,6 +234,7 @@
       { "item": "decoy_elfa", "prob": 5 },
       { "item": "fuel_tin", "prob": 4 },
       { "item": "spray_can", "prob": 2 },
+      { "item": "insecticide_spray_can", "prob": 10, "charges": [ 0, 2 ] },
```

```suggestion
      { "item": "insecticide_spray_can", "prob": 5, "charges": [ 0, 2 ] },
```
I tried to ~halve the amount of items in `SUS_trash_trashcan_half` compared to `SUS_trash_trashcan_full` so please follow that.